maintenance/ebpf/ebpf-and-eks.md

```diff
@@ -40,8 +40,6 @@ EKS is Amazon's managed Kubernetes offering.
 
 ### How to
 
-- [Create an EKS cluster with a recent enough kernel](#create-an-eks-cluster-with-a-recent-enough-kernel)
-
 #### Create an EKS cluster with a recent enough kernel
```

Comment:
Not sure what "recent enough" means...

Why not just "Create an EKS cluster"? The requirement is a kernel version, which we explain so not sure we need to qualify this in the title.

Reply:
How about:
```suggestion
#### Create an eBPF compatible EKS cluster 
```